tests/unit/_internal/models/test_model.py

```diff
@@ -59,7 +59,9 @@
     batchable: false
   classify:
     batchable: true
-    batch_dim: 0
+    batch_dim:
```

Comment:
shall we have
```
input_batch_dim: 0
output_batch_dim: 0
```
here? @parano

Reply:
yes, I thought you had some refactoring work for batch_dim that will do this?

Reply:
@parano yeah but I need an extra approval since it's simplified a lot.

Reply:
will do that in a small refactoring PR